src/com/severalcircles/colors/commands/CommandSpringaling.java

```diff
@@ -0,0 +1,33 @@
+package com.severalcircles.colors.commands;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+public class CommandSpringaling implements CommandExecutor {
+    @Override
+    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] args) {
+//        boolean isPlayer = true;
+        Player player = (Player) commandSender;
+        Player target = Bukkit.getPlayer(args[0]);
+        if (target == null) {
+            player.sendMessage("Player not found");
+            System.out.println("Player not found");
+            return true;
+        }
+        if (!player.hasPermission("com.severalcircles.colors.spring")) {
+            player.sendMessage(ChatColor.YELLOW + "Argumento incorecto");
```

Comment:
This is inconsistent with the rest of the permission denied messages. I would recommend making this the "Ah Ah Ah!" message.

Reply:
The "Ah Ah Ah!" is a reference to to Dennis Nedry which is what the command is referencing. Maybe he should fix that one because it wouldn't make sense here.

Reply:
The idea we had was to have the "Ah ah ah!" on every command.

Reply:
I was referring to the permission, not the message. Apologies.